# **Estimation And Confidence Intervals**

Background:

In quality control processes, especially when dealing with high-value items, destructive sampling is a necessary but costly method to ensure product quality. The test to determine whether an item meets the quality standards destroys the item, leading to the requirement of small sample sizes due to cost constraints.

Scenario:

A manufacturer of print-heads for personal computers is interested in estimating the mean durability of their print-heads in terms of the number of characters printed before failure. To assess this, the manufacturer conducts a study on a small sample of print-heads due to the destructive nature of the testing process.

Data:

A total of 15 print-heads were randomly selected and tested until failure. The durability of each print-head (in millions of characters) was recorded as follows:
1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29

**a. Build 99% Confidence Interval Using Sample Standard Deviation**

Assuming the sample is representative of the population, construct a 99% confidence interval for the mean number of characters printed before the print-head fails using the sample standard deviation. Explain the steps you take and the rationale behind using the t-distribution for this task

In [2]:
# Import required libraries
import numpy as np
import scipy.stats as stats

# Sample data
data = np.array([1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29])

# Step 1: Calculate the Sample Mean
sample_mean = np.mean(data)

# Step 2: Calculate the Sample Standard Deviation
sample_std = np.std(data, ddof=1)  # ddof=1 for sample standard deviation

# Step 3: Determine the Sample Size
n = len(data)

# Step 4: Calculate Degrees of Freedom
df = n - 1

# Step 5: Find the Critical Value (t) for 99% Confidence Interval
confidence_level = 0.99
alpha = 1 - confidence_level
t = stats.t.ppf(1 - alpha/2, df)

# Step 6: Calculate the Standard Error of the Mean (SE)
se = sample_std / np.sqrt(n)

# Step 7: Compute the Margin of Error (ME)
me = t * se

# Step 8: Determine the Confidence Interval
ci_lower = sample_mean - me
ci_upper = sample_mean + me

print(f"Sample Mean: {sample_mean:.3f}")
print(f"Sample Standard Deviation: {sample_std:.3f}")
print(f"Standard Error: {se:.3f}")
print(f"Degrees of Freedom: {df}")
print(f"Critical Value (t): {t:.3f}")
print(f"Margin of Error: {me:.3f}")
print(f"99% Confidence Interval: ({ci_lower:.3f}, {ci_upper:.3f})")

Sample Mean: 1.239
Sample Standard Deviation: 0.193
Standard Error: 0.050
Degrees of Freedom: 14
Critical Value (t): 2.977
Margin of Error: 0.148
99% Confidence Interval: (1.090, 1.387)


Interpretation:

The 99% confidence interval for the mean number of characters printed before the print-head fails is approximately (1.090, 1.387). This means we are 99% confident that the true population mean lies within this interval.


Conclusion:

Using the t-distribution to construct a confidence interval is crucial when dealing with small sample sizes and unknown population standard deviations. It accounts for the increased uncertainty and provides a more reliable interval estimate compared to the normal distribution. By following these steps and implementing the process in Python, we ensure that our interval estimates are accurate and meaningful.

**b. Build 99% Confidence Interval Using Known Population Standard Deviation**

If it were known that the population standard deviation is 0.2 million characters, construct a 99% confidence interval for the mean number of characters printed before failure.

In [3]:
# Sample data
data = np.array([1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29])

# Step 1: Calculate the Sample Mean
sample_mean = np.mean(data)

# Step 2: Population Standard Deviation
population_std = 0.2  # Given population standard deviation

# Step 3: Determine the Sample Size
n = len(data)

# Step 4: Find the Critical Value (z) for 99% Confidence Interval
confidence_level = 0.99
alpha = 1 - confidence_level
z = stats.norm.ppf(1 - alpha/2)

# Step 5: Calculate the Standard Error of the Mean (SE)
se = population_std / np.sqrt(n)

# Step 6: Compute the Margin of Error (ME)
me = z * se

# Step 7: Determine the Confidence Interval
ci_lower = sample_mean - me
ci_upper = sample_mean + me

print(f"Sample Mean: {sample_mean:.3f}")
print(f"Population Standard Deviation: {population_std:.3f}")
print(f"Standard Error: {se:.3f}")
print(f"Critical Value (z): {z:.3f}")
print(f"Margin of Error: {me:.3f}")
print(f"99% Confidence Interval: ({ci_lower:.3f}, {ci_upper:.3f})")

Sample Mean: 1.239
Population Standard Deviation: 0.200
Standard Error: 0.052
Critical Value (z): 2.576
Margin of Error: 0.133
99% Confidence Interval: (1.106, 1.372)


Interpretation:

The 99% confidence interval for the mean number of characters printed before the print-head fails, given that the population standard deviation is 0.2 million characters, is approximately (1.106, 1.372). This means we are 99% confident that the true population mean lies within this interval.

Conclusion:

Using the z-distribution when the population standard deviation is known provides a more precise confidence interval. The critical value from the normal distribution is used to calculate the margin of error, leading to an accurate estimate of the population mean. By following these steps and implementing the process in Python, we ensure that our interval estimates are reliable and meaningful.






